osu.Game.Rulesets.Taiko/TaikoSkinComponents.cs

```diff
@@ -11,6 +11,7 @@ public enum TaikoSkinComponents
         DrumRollBody,
         DrumRollTick,
         Swell,
+        SwellCirclePiece,
```

Comment:
Can `Swell` be renamed to `SwellBody` or something? Having `Swell` next to `SwellCirclePiece` just looks confusing to me.